### DISCUSSION TITLE
What about creating a Vite plugin for Kubb?

### DISCUSSION THREAD

**Discussion by gunta** [kubb-labs/kubb#762] [Ideas]
It would be nice to have the inputs watched from Vite.

**Comment by stijnvanhulle** [comment]
@gunta I Love the idea, I will have a look at creating this 💪🏽.

**Reply by stijnvanhulle** [reply]
You can try it out with: [unplugin-kubb](https://www.npmjs.com/package/unplugin-kubb)

**Reply by nnecec** [reply]
@stijnvanhulle the `unplugin-kubb` seems to not support multiple configs.
